src/index.js

```diff
@@ -1,11 +1,58 @@
-import _ from "lodash";
+import './style.css';
 
-function component () {
-  const element = document.createElement('div');
+const element = document.querySelector('#todolist');
+const toDoList = [
+  {
+    description: 'wash the dishes',
+    completed: false,
+    index: 0,
+  },
+  {
+    description: 'complete To Do list project',
+    completed: false,
+    index: 1,
+  },
+];
 
-  element.innerHTML = _.join(['hello', 'webpack'], ' ');
+const getToDoList = () => {
+  if (toDoList === null) {
+    return [];
+  }
+  return toDoList;
+};
 
-  return element;
-}
+// eslint-disable-next-line consistent-return
+const printList = () => {
+  const list = toDoList;
+  let tasks = '';
+  for (let i = 0; i < list.length; i += 1) {
+    // eslint-disable-next-line no-loop-func

```

Comment:
- [ ]  [optional] It's not good practice to disable linter checks, kindly try and fix the error first before disabling it.